.github/workflows/ci-ruby.yml

```diff
@@ -0,0 +1,13 @@
+name: Ruby linting and tests
+on:
+  push:
+    paths: ['**/*.rb', '**/*.rake', '.github/workflows/ci-ruby.yml']
```

Comment:
Looks good. There are a few more Ruby files to add: `.ruby-version`, `.rspec`, `.rubocop.yml`, `Rakefile`, `Gemfile`.

Reply:
Aha, thanks

Reply:
I'm happy if we want to push this stuff down into a subdirectory (hard to name though, `scripts` or `ruby` both feel too generic, though we have directories for `docker` and `terraform`)

Reply:
Yeah, I wondered about moving stuff around but thought it's probably more robust to have the job not care where the source files are.